editoast/src/main.rs

```diff
@@ -266,9 +267,22 @@ async fn run() -> Result<(), Box<dyn Error + Send + Sync>> {
                     .map_err(Into::into)
             }
         },
+        Commands::Healthcheck => healthcheck_cmd(db_pool.into(), redis_config).await,
     }
 }
 
+async fn healthcheck_cmd(
+    db_pool: Arc<DbConnectionPoolV2>,
+    redis_config: RedisConfig,
+) -> Result<(), Box<dyn Error + Send + Sync>> {
+    let redis = RedisClient::new(redis_config).unwrap();
+    check_health(db_pool.into(), redis.into())
+        .await
+        .map_err(|e| CliError::new(1, format!("❌ healthcheck failed: {0}", e)))?;
+    println!("❎ Healthcheck passed");
```

Comment:
Green cross is leaving a bitter-sweet taste in my mouth :P 

(Do we really want to log each and every healthcheck request?)

Reply:
It will not be logged in the output, it's executed by k8s directly, we do not have this in the log iirc.